spec/models/season/phase_switcher_spec.rb

```diff
@@ -4,32 +4,61 @@
 context 'when switching phases' do
   subject { Season.current }
 
-  describe '#fake_application_phase' do
-    it 'timeshifts the application phase to today' do
-      Timecop.travel(Season.current.applications_close_at - 1.month) do
-        Season::PhaseSwitcher.fake_application_phase
-        expect(subject).to be_application_period
+  describe 'destined' do
+
+    it 'sends the params to a phase setting method' do
```

Comment:
Sorry, I don't get what this spec is supposed to do. Why does calling the `PhaseSwitcher` have an impact on weather there is a current season or not?